src/components/Dashboard/widgets/CommandButtonWidget.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { CommandButtonWidget } from './CommandButtonWidget';
+
+describe('Command Button Widget', () => {
+  it('renders without crashing', () => {
+    shallow(<CommandButtonWidget />);
+  });
+
+  it('should have default props', () => {
+    const { defaultProps } = CommandButtonWidget;
+    expect(defaultProps.mode).toBe('library');
+    expect(defaultProps.device).toBe('');
+    expect(defaultProps.command).toBe('');
+    expect(defaultProps.params.value).toBeNull();
+  });
+
+  it('shows `Device Name` without command', () => {
+    const device = 'sys/test/1';
+    const wrapper = shallow(<CommandButtonWidget device={device} mode="library" />);
+    expect(wrapper).toHaveText('Set ' + device + ' command');
```

Comment:
This should use string templates rather than concatenation, currently it throws an esLint error.
```
expect(wrapper).toHaveText(`Set ${device} command`);
```

Reply:
fixed